EnglishFilter/sections/specific.txt

```diff
@@ -294,6 +294,7 @@ linkjust.com,updatesmarugujarat.in,mexashare.com,links4u.me,linku.us,skips.link,
 !
 !
 !
+fashionunited.*##.adunitContainer:upward(3)
```

Comment:
@Morku It has to be in `css_extended` file and written via `#?#` - will move it.
As I can see, it works fine on the website.
Btw, we have added comments at the headers of the each file - please read them.